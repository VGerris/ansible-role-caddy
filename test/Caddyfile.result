{
  admin off
}


example.com {
  
  @allowlist {
    remote_ip 8.8.8.8/32
  }

  @not_allowlist {
    not remote_ip 8.8.8.8/32
  }

  reverse_proxy @allowlist 192.168.50.2
  respond @not_allowlist 404
  
  
}
