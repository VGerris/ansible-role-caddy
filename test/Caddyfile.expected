{
  admin off
}


example.com {
  @allowlist {
    remote_ip  8.8.8.8/32
  }

  @not_allowlist {
    not remote_ip  8.8.8.8/32
  }
  
  
  handle  {
    reverse_proxy @allowlist 192.168.50.2
    respond @not_allowlist 404
  }
  
  }


http://example.com:8080, http://example.com:1337 {
  redir http://example.com{uri} permanent
}

